doc/help/embed.md

```diff
@@ -0,0 +1,271 @@
+# Introduction
+
+# Login
+
+![Login](login.png)
+
+# User Management
+
+Click on the User Management Tab to see the User Management
+screen. This allows you to search for any user in the system by their
+registered email address and add them to any programmes or projects
+that you are a Programme Manager or Project Manager for.
+
+![User Management](user-management.png)
+
+## Editing Users
+
+If you are an administrator you can set the overall role for a user as
+User, Admin or Super Admin. If you are not an administrator but are a
+Programme or Project Manager, then you can make a user into a
+Programme Manager, Project Manager or user on any Programme or Project
+you manage.
+
+All managers and administrators have rights to edit data in programmes
+or projects they have been added to. These rights cascade down into
+the projects, properties, devices etc contained in the Programme or
+Project they are a manager for.
+
+All users have the right to view private data in the programmes or
+projects they have been added to.
+
+![Editing User](editing-user.png)
+
+Here is an example of making a user a Programme Manager.
+
+![Make user a programme manager](user-as-programme-manager.png)
+
+This is what it looks like when you have successfully edited a user.
+
+![Successful Editing](successful-user-editing.png)
+
+# Programmes
```

Comment:
Need to explain the concepts of programme, project, device, etc before this point in the doc